docker-bake.hcl

```diff
@@ -0,0 +1,22 @@
+
+variable "RELEASE" {
+    default = "v1.0.0"
+}
+
+variable "REGISTRY" {
+    default = "ghcr.io/SecretiveShell/MCP-Bridge/"
```

Comment:
I am fairly sure that the docker registry will break if you do not have all lowercase